src/styles/RadioButtonPage.scss

```diff
@@ -0,0 +1,2 @@
+@import "@material/radio/dist/mdc.radio";
+@import "@material/form-field/dist/mdc.form-field";
```

Comment:
add a little bit more space between the "Radio 1" and "Radio 2" so the ripples do not overlap